src/core/caniuse.js

```diff
@@ -98,48 +101,85 @@ export async function run(conf) {
   }
 }
 
+function handleError(err, options, featureURL) {
+  const msg = `Failed to retrieve feature "${options.feature}".`;
+  const hint = docLink`Please check the feature key on [caniuse.com](https://caniuse.com) and update ${"[caniuse]"}.`;
+  showError(msg, name, { hint });
+  console.error(err);
+  return html`<a href="${featureURL}">caniuse.com</a>`;
+}
+
 /**
  * returns normalized `conf.caniuse` configuration
  * @param {Object} conf   configuration settings
  */
-function getNormalizedConf(conf) {
-  const DEFAULTS = { versions: 4, removeOnSave: false };
+function normalizeCaniuseConf(conf) {
+  const DEFAULTS = { removeOnSave: false, browsers: [] };
   if (typeof conf.caniuse === "string") {
-    return { feature: conf.caniuse, ...DEFAULTS };
+    conf.caniuse = { feature: conf.caniuse, ...DEFAULTS };
+    return;
   }
-  const caniuseConf = { ...DEFAULTS, ...conf.caniuse };
-  const { browsers } = caniuseConf;
-  if (Array.isArray(browsers)) {
-    const invalidBrowsers = browsers.filter(browser => !BROWSERS.has(browser));
-    if (invalidBrowsers.length) {
-      const names = codedJoinAnd(invalidBrowsers, { quotes: true });
-      const msg = docLink`Invalid browser(s): (${names}) in the \`browser\` property of ${"[caniuse]"}.`;
-      showWarning(msg, name);
-    }
+  conf.caniuse = { ...DEFAULTS, ...conf.caniuse };
+}
+
+function validateBrowsers({ caniuse }) {
+  const { browsers } = caniuse;
+  const invalidBrowsers = browsers.filter(browser => !BROWSERS.has(browser));
+  if (invalidBrowsers.length) {
+    const names = codedJoinAnd(invalidBrowsers, { quotes: true });
+    const msg = docLink`Invalid browser(s): (${names}) in the \`browser\` property of ${"[caniuse]"}.`;
+    showWarning(msg, name);
   }
-  return caniuseConf;
+}
+
+async function processJson(json, { feature }) {
+  const results = json.result;
+  const out = results.map(({ browser: browserId, version, caniuse }) => {
+    const { name } = BROWSERS.get(browserId);
+    const versionLong = version ? ` version ${version}` : "";
+    const browserName = `${name}${versionLong}`;
+    const supportLevel = statToText.get(caniuse);
+    const ariaLabel = `${feature} is ${supportLevel} in ${browserName}.`;
```

Comment:
Firefox a11y panel shows it like this:
![image](https://user-images.githubusercontent.com/8426945/157854029-623d57cc-a810-42c1-bf23-e79ddacb403f.png)

Should test it out with Voiceover/NVDA